app/controllers/submissions_controller.rb

```diff
@@ -114,6 +114,13 @@ def repo_browser
       end
     end
 
+    last_rev = @grouping.submissions
+    @last_submission = nil
+    if !last_rev.empty?
+      selected = @revisions_history.select {|rev| rev[:num] == last_rev.last.revision_number}
```

Comment:
Line is too long. [93/80]<br>Space between { and | missing.<br>Space missing inside }.